docker/Parsl/test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+import parsl
+import os
+from parsl.app.app import python_app, bash_app
+from parsl.configs.local_threads import config
+
+#parsl.set_stream_logger() # <-- log everything to stdout
+
+print(parsl.__version__)
+
+parsl.load(config)
+
+@python_app
+def hello ():
+    return 'Hello World from Python!'
+
+print(hello().result())
+
+@bash_app
+def echo_hello(stdout='/tmp/echo-hello.stdout', stderr='/tmp/echo-hello.stderr'):
+    return 'echo "Hello World!"'
+
+echo_hello().result()
+
+with open('/tmp/echo-hello.stdout', 'r') as f:
+    print(f.read())
```

Comment:
Might be worthwhile to check that `f.read() == "Hello World!"` and exit with a non-zero exit code if not.  Same for the `hello().result()` above.  That way the `docker run -ti parsl` line in the CI will fail if this sanity test fails.

Reply:
The purpose was to do a basic sanity check. If things were broken, lots will not work before there would be a subtle difference between output strings.

That said, it can't hurt to add it.